source/greentea_test_env.cpp

```diff
@@ -72,7 +72,7 @@ void _GREENTEA_SETUP_COMMON(const int timeout, const char *host_test_name, char
 
     while (1) {
         greentea_parse_kv(_key, buffer, sizeof(_key), size);
-        greentea_write_string("mbedmbedmbedmbedmbedmbedmbedmbed\r\n");
+        greentea_write_string("greenteagreenteagreenteagreentea\r\n");
```

Comment:
`mbedmbedmbedmbedmbedmbedmbedmbed` is a magic string recognised by the mbed-greentea host runner. Changing this will break compatibility. For now we need to keep the old string.